### PR TITLE
added OSINT topic

### PR DESCRIPTION
**Pull request by mxrch**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> There is a lot of OSINT repos now on Github, and I think that we need to have a proper topic for this today.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/osint/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+topic: osint
+related: malfrats
+aliases: investigation
+display_name: OSINT
+logo: osint.png
+short_description: All projects related to OSINT.
+url: https://en.wikipedia.org/wiki/Open-source_intelligence
+wikipedia_url: https://en.wikipedia.org/wiki/Org-mode
```
Comment (MikeMcQuaid) [inline topics/osint/index.md:9]: Whoops this doesn't look quite right!

File: topics/osint/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+topic: osint
+related: malfrats
+aliases: investigation
+display_name: OSINT
+logo: osint.png
```
Comment (MikeMcQuaid) [inline topics/osint/index.md:6]: What is "malfrats industries"?

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks for the PR! A few things need addressed.

**Review by dzimidrol5 — APPROVED** [review]

**Review by dzimidrol5 — APPROVED** [review]

File: topics/osint/index.md (hunk 1 above)
Comment (mxrch) [reply topics/osint/index.md]: Oops, I fixed it :)

File: topics/osint/index.md (hunk 2 above)
Comment (mxrch) [reply topics/osint/index.md:5]: It is a collective of developers of well used open-source OSINT tools !

Comment (nholden) [reply topics/osint/index.md:5]: Is there an image that might better represent OSINT? As someone new to the topic, it's not clear to me how that logo connects to OSINT.

**Review by Vannsdok — CHANGES_REQUESTED** [review]
script/cibuild

**Comment by Vannsdok** [comment]
mxrch:main

**Review by Vannsdok — CHANGES_REQUESTED** [review]
topics/osint/osint.png

**Review by Vannsdok — CHANGES_REQUESTED** [review]
topics/osint/osint.png

**Review by Vannsdok — CHANGES_REQUESTED** [review]
*  @github/communities-oss-reviewers

File: topics/osint/index.md (hunk 1 above)
Comment (Vannsdok) [reply topics/osint/index.md]: *2394

**Review by Vannsdok — CHANGES_REQUESTED** [review]

**Comment by kenyonj** [comment]
Hi @mxrch, please re-open this PR with a new image, or feel free to remove the image. Thank you.

**Pull request closed without merging**